### PR TITLE
Fix logging 

### LINKED ISSUE #2: Logging not working
It seems like the logging option isn't working as expected.   My TypeScript code looks like this:

```
import exec from "async-exec"

async function foo() {
    const prepare = await exec('npm run prepare', true)
    ...
}
...

foo()
```
After the await, `prepare` does contain the results of running the command, but they never get logged to the console. Inside your code, `printLines(data)` never gets called.   

The async part seems to be working, just not the logging.    I'm running on node v6.10.3 on Windows 10.

### PR DESCRIPTION
**Pull request by BurtHarris**
Fixes issue #2 (logging not working) by attaching data events on `process.stdout` rather than `process`.

I've proposed a patch version bump too, not sure how you handle this.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,2 @@
 node_modules
-*.js
-*.js.map
-*.d.ts
```
Comment (coldpour) [inline .gitignore]: I am not a fan of checking compiled files into source control. Do you need this for some reason?

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export default function (command: string, logOutput?: boolean): Promise<string>;
```
Comment (coldpour) [inline index.d.ts]: I am not a fan of checking compiled files into source control. Do you need this for some reason?

File: index.ts [hunk 3]
```diff
@@ -17,7 +17,7 @@ export default function(command: string, logOutput: boolean = false): Promise<st
     });
 
     if (logOutput) {
-      process.on('data', (data: Object) => printLines(data));
+      process.stdout.on('data', (data: Object) => printLines(data));
```
Comment (coldpour) [inline index.ts]: It seems to be working for me on my mac running node 7.2.0 without this change. Why do you suspect this change is needed?

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "async-exec",
-  "version": "1.0.3",
+  "version": "1.0.5",
```
Comment (coldpour) [inline package.json]: I appreciate the suggestion of a version bump, but why up to 5 from 3?

File: index.ts (hunk 3 above)
Comment (coldpour) [inline index.ts]: ah, i've got it reproduced now. sorry about that. your fix works great.

File: .gitignore (hunk 1 above)
Comment (BurtHarris) [inline .gitignore]: Hmm, no good reason.   I generally don't check in built files either.   Perhaps I changed it temporarily to compare generated code and forgot to back it out.

File: package.json (hunk 4 above)
Comment (BurtHarris) [inline package.json]: again, not sure...  I'll fix that too.

**Comment by coldpour** [comment]
I just merged this fix into master with some tests. You can get it in v1.0.4. Thanks for the fix Burt!

File: .gitignore (hunk 1 above)
Comment (BurtHarris) [inline .gitignore]: As noted in the commit comment, I did that to allow me to publish a private build it from my GitHub fork before the PR was merged.   It should not go into your branch.   I'll fix it.

File: package.json (hunk 4 above)
Comment (BurtHarris) [inline package.json]: I think this was part of my private-source versioning stuff, I couldn't get the install-from-GitHub option to work the first time I tried it.    Again, that can be fixed, will do.

Comment (BurtHarris) [inline package.json]: OK, if you've already merged it, I won't make the changes myself.

**Pull request closed without merging**